docs/CHANGELOG.md

```diff
@@ -23,6 +23,10 @@ title: Changelog
 
     *Dany Marcoux*
 
+* Add components to `rails stats` task.
```

Comment:
I believe this PR should be rebased to add this above the release I just shipped today ❤️

Reply:
Done 🙂